nautilus_core/core/Cargo.toml

```diff
@@ -20,6 +20,7 @@ serde_json = { workspace = true }
 ustr = { workspace = true }
 uuid = { workspace = true }
 heck = "0.5.0"
+pretty_assertions = {version = "1.4.0"}
```

Comment:
We could even make this a workspace level dev dependency?

Reply:
Yes, we cool. It will be probably used in other crates also